data/build.gradle

```diff
@@ -0,0 +1,31 @@
+apply plugin: 'com.android.library'
+
+apply from: './gradleFiles/dependenciesData.gradle'
+
+android {
+    compileSdkVersion 28
+
+    defaultConfig {
+        minSdkVersion 21
+        targetSdkVersion 28
+        versionCode 1
+        versionName "1.0"
+
+        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+        }
+    }
+}
+
+dependencies {
+    implementation fileTree(dir: 'libs', include: ['*.jar'])
```

Comment:
@OscarGonzalezLesaga Maybe we can also remove this line, because is not used

Reply:
Yeap, it's true